dashboard/app/models/regional_partner.rb

```diff
@@ -59,6 +57,14 @@ def program_manager=(program_manager_id)
     )
   end
 
+  def contact
+    User.find(read_attribute(contact_id)) || program_managers.first
+  end
+
+  def contact=(user)
+    write_attribute(:contact_id, user.id)
+  end
```

Comment:
What about nil? Do we ever clear the contact?

Reply:
good point